### ISSUE TITLE
Path issue with git hooks

### ISSUE THREAD

**Issue by Snaver** [phpro/grumphp#447]
| Q               | A
| --------------- | ---
| Version         | 0.13.1
| Bug?            | yes
| New feature?    | no
| Question?       | yes
| Documentation?  |no
| Related tickets | 

Seems there's an error with git hooks on windows? Are slashes being stripped?

Same error using both Sourcetree and gitkraken:

```git -c diff.mnemonicprefix=false -c core.quotepath=false commit -q -F C:\Users\XXX\AppData\Local\Temp\itngomnn.xpn
.git/hooks/pre-commit: line 12: exec: C:UsersXXXRepositoriesXXX-laravelvendorbingrumphp: not found
Completed with errors, see above.
```

Can run `.\vendor\bin\grumphp run` from Powershell with no issues.

* PHP 7.2.0
* Windows 10 
* git version 2.15.1.windows.2

**Comment by pedro-p** [comment]
Same issue for me.
Adding double quotation marks to pre-commit seems to fix this on windows: https://github.com/phpro/grumphp/pull/458/commits/c774186c4d47e9c44df75484d30c65d3d7a64894

**Comment by Ainaras** [comment]
Same issue for me too. I can't start using this in my org, until this is fixed. 
Patch from pedro-p does the thing. Can't wait until this patch is merged.

**Comment by veewee** [comment]
Can you confirm that #463 fixes the issue?

**Comment by Snaver** [comment]
Just pulled in the fix-force-unix-hooks branch, seemed to fix the issue for me 👍

**Comment by veewee** [comment]
Thanks for checking @Snaver ! Fixed in #463
